src/olympia/reviewers/utils.py

```diff
@@ -696,6 +746,11 @@ def process_sandbox(self):
         self.set_files(amo.STATUS_DISABLED, self.files,
                        hide_disabled_file=True)
 
+        # Unset needs_human_review on the latest version - it's the only
+        # version we can be certain that the reviewer looked at.
+        if self.request:
```

Comment:
mmm, I know we test on `self.request` as a way of determining if it's an auto approval by the cron or a manual one, but it's a bit magic sauce, especially when there no notes in the method about the logic.  I wonder if we should just set a `self.is_not_cron` (with a better name) to be explicit about it.

Reply:
397194b60bd0a50c0d3982a4cd01c8e9717b51b5